layouts/release/9.1/section.html

```diff
@@ -0,0 +1,47 @@
+{{ define "head-suffix" }}
+<link href="https://fonts.googleapis.com/css2?family=Exo:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+{{ end }}
+
+{{ define "header"}}
+  <a class="sr-only" href="#content">Skip to main content</a>
+  <header class="header-wrapper{{- with .Params.header_wrapper_class | default .Site.Params.header_wrapper_class }} {{ . }}{{end}}" id="header-wrapper">
+    <!---Navbar--->
+    {{ partial "navbar.html" . }}
+    <!---Navbar end --->
+
+    {{ if isset .Page.Params "headline" }}
+    <div class="jakartaee-9-jumbotron-wrapper">
+      <video autoplay muted loop id="jakartaee-9-video-background">
+        <source src="/release/9/videos/header-video.mp4" type="video/mp4">
+      </video>
+      <div class="jumbotron featured-jumbotron featured-jumbotron-download vertical-align flex-column margin-bottom-0">
+        <div class="jakartaee-9-flex-grow-container vertical-align flex-wrap">
+          <div class="col-md-10 flex-column">
+            <p style="align-self: flex-start;"><span class="p text-primary">version 9.1</span> new release</p>
+            <h1>Jakarta&nbsp;EE</h1>
+          </div>
+          <div class="col-md-2 col-md-offset-2 vertical-align">
+            <span class="jumbotron-right-content"><i>9.1</i></span>
+          </div>
+          <div class="col-md-9 col-md-offset-1 vertical-align" style="text-align: left;">
+            <p><span class="p text-primary">An open, community-driven platform</span> <br> for future innovation of <br> enterprise Java technologies</p>
+          </div>
+        </div>
+  
+        <div class="container jakartaee-9-jumbotron-button-container">
+          <ul class="list-inline buttons">
+            <li class="vcenter"><a class="btn btn-primary vertical-align" href="/compatibility/">DOWNLOAD <br/> COMPATIBLE PRODUCTS</a></li>
+            <li class="vcenter"><a class="btn btn-primary vertical-align" href="/specifications/">SPECIFICATIONS</a></li>
+          </ul>
+
+          <a id="text-white" href="/release/9">PREVIOUS RELEASE: JAKARTA EE 9</a>
```

Comment:
Not a deal breaker, but its better to just change how the chars are displayed rather than uppercase the letters. Sometimes accessibility tools will read all uppercase text strangely or with different emphasis. I believe the `uppercase` class does this in our framework

Reply:
Thanks much @autumnfound 
Fixed here 439c9c2f

Reply:
btw, @autumnfound and team,  feel free to push to this PR.

I opened the PR for a demo I made at last week's JakartaEE Marketing Committee call, now the PR had (and probably will have) minor updates like the ones Karen requested [early today](https://github.com/jakartaee/jakarta.ee/issues/1179#issuecomment-843170549) 

Issues like the one [reported by Amelia](https://github.com/jakartaee/jakarta.ee/issues/1165#issuecomment-843573383) seem to require more internal troubleshooting from you and Team. Thank you in advance for the help.